modules/apps/frontend-js/frontend-js-web/test/liferay/util/store/get_store_value.es.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+'use strict';
+
+import getStoreValue from '../../../../src/main/resources/META-INF/resources/liferay/util/store/get_store_value.es';
+
+describe('Liferay.Util.Store.getStoreValue', () => {
+	beforeEach(() => {
```

Comment:
All values that are mocked in `beforeEach` should be stored in a buffer and restored in `afterEach`. To achieve this, we only need to backup and restore `Liferay`:

```
describe('Liferay.Util.Store.getStoreValue', () => {
    let globalLiferay;

    afterEach(() => {
        Liferay = globalLiferay;
    }

    beforeEach(() => {
        globalLiferay = Liferay;

        Liferay = {
             authToken: 'abcd',
             ...
        }
    }
```